prime-router/metadata/hl7_mapping/resources/ServiceRequest/OBRExtension.yml

```diff
@@ -28,8 +28,9 @@ elements:
   - name: collector-identifier
     resource: '%resource.extension.where(url = "obr-12-danger-code").value'
     schema: ../../datatypes/codeableConcept/CWE
+    resourceIndex: cweIndex
```

Comment:
Doesn't look like this is repeatable?

Reply:
yep good call. Reviewed the repeatable fields and set them correctly

Reply:
Might gotten missed here on your second pass